pgcommitfest/commitfest/models.py

```diff
@@ -102,11 +102,37 @@ def __str__(self):
         return self.version
 
 
+class ColorField(models.CharField):
+    """
+    A small wrapper around a CharField that can hold a #RRGGBB color code. The
+    primary reason to have this wrapper class is so that the TagAdmin class can
+    explicitly key off of it to inject a color picker in the admin interface.
+    """
+
+    def __init__(self, *args, **kwargs):
+        kwargs["max_length"] = 7  # for `#RRGGBB` format
+        super().__init__(*args, **kwargs)
+
+
+class Tag(models.Model):
```

Comment:
Can you add some default entries for this to the dummy database dump: https://github.com/postgres/pgcommitfest#regenerating-the-database-dump-files Or maybe even add a few to that we know we definitely want to the migration.